.travis.yml

```diff
@@ -2,8 +2,6 @@ language: python
 sudo: false
 cache:
   - pip
-python:
-  - 3.6
 env:
   global:
     - ASYNC_TEST_TIMEOUT=15
```

Comment:
I recall this being irrelevant also - do you know about this @minrk?

Reply:
I believe you're right @consideRatio, no need for ASYNC_TEST_TIMEOUT anymore (found this as a reference https://github.com/jupyterhub/jupyterhub/pull/2812)
I'll remove it :).

Reply:
This was outside the scope of this PR so thanks for considering it, it was helpful to me to get confirmation on this!